modules/ROOT/pages/index.adoc

```diff
@@ -180,69 +305,65 @@ If your organization changes its subscription model from a Titanium to a Gold or
 
 4+| *Alerts*
 
-|Basic alerts for apps, servers, and APIs
+|xref:basic-alerts.adoc[Basic alerts] for apps, servers, and APIs
 |Yes
 |Yes
 |Yes, with xref:performance-and-impact.adoc#data-retention-and-aggregation[higher limits]
 
-|Advanced alerts
+|xref:advanced-alerts.adoc[Advanced alerts]
 |No
 |No
 |Yes
 
 4+| *Log Management*
 
-|Log search
+|xref:log-search-query-syntax.adoc[Log search]
 |No
 |No
 |Yes
 
-|Log points
+|xref:log-points.adoc[Log points]
 |No
 |No
 |Yes
 
-|Raw data
+|xref:raw-data.adoc[Raw data]
 |No
 |No
 |Yes
 
 4+| *Additional Anypoint Monitoring features*
 
-|Anypoint Monitoring Archive API
+|xref:api-analytics-dashboard.adoc#anypoint-monitoring-archive-api[Anypoint Monitoring Archive API]
 |No
 |No
 |Yes
 
-|xref:api-functional-monitoring[API Functional Monitoring] +
+|<<functional-monitoring, API Functional Monitoring>>
 |Yes
 |No
 |Yes
 
-|Reports
-|No
-|No
-|Yes
-
-|Telemetry Exporter
+|xref:reports.adoc[Reports]
 |No
 |No
 |Yes
 
 |===
 
-NOTE: Anypoint Visualizer application network integration refers to an Anypoint Visualizer widget within Anypoint Monitoring. Anypoint Visualizer as a product outside of Anypoint Monitoring is available to organizations regardless of pricing plan. 
+[[note-visualizer]]
+NOTE: Anypoint Visualizer app network integration refers to an Anypoint Visualizer widget within Anypoint Monitoring. Anypoint Visualizer as a product outside of Anypoint Monitoring is available to organizations regardless of pricing plan. 
 
 
 == Use Cases
 
 The following use cases provide examples of how you can use Anypoint Monitoring.
 
 * Understand performance-related KPIs +
-DevOps engineer Gary maintains a series of APIs related to a transaction processing system. Because the TPS requires swift response time, Gary uses API response time dashboards and alerts to send an email when there is a bottleneck. After gathering some data, Gary can then size his applications.
+DevOps engineer Gary maintains a series of APIs related to a transaction processing system. Because the TPS requires swift response time, Gary uses API response time dashboards and alerts to send an email when there is a bottleneck. After gathering some data, Gary can then size his apps.
```

Comment:
```suggestion
DevOps engineer Gary maintains a series of APIs related to a transaction processing system. Because the TPS requires swift response, Gary uses API response-time dashboards and alerts to send an email when a bottleneck occurs. After gathering some data, Gary can then size his apps.
```